code/datums/quirks/negative_quirks/negative_quirks.dm

```diff
@@ -1,4 +1,5 @@
 //predominantly negative traits
+#define MOOD_CATEGORY_PHOTOPHOBIA "photophobia"
```

Comment:
Can we put this define and its undef before and after the photophobia code? Also we probably should start using new files, this one is 1500 lines long now.